lightwood/encoders/time_series/helpers/rnn_helpers.py

```diff
@@ -2,58 +2,84 @@
 
 import torch
 import torch.nn as nn
-import torch.nn.functional as F
 
-import numpy as np
 
-
-def tensor_from_series(series, device):
-    if type(series) != type([]):
-        series = str(series).split(' ')
+def tensor_from_series(series, device, n_dims=None, pad_value=0.0, max_len=64):
```

Comment:
Are the optional arguments ever not set explicitly ? If not, they shouldn't be optional.